### PR TITLE
Refactoring SABER CI testing

### PR DESCRIPTION
**Pull request by benjaminmenetrier**
Companion PR of https://github.com/JCSDA-internal/saber/pull/358.

### CODE REVIEW

File: test/testinput/dirac_soca_mask.yml [hunk 1]
```diff
@@ -16,15 +16,13 @@ background error:
...
   bump:
-    verbosity: main
     datadir: data_generated/static_socaerror_init
     method: cor
     strategy: specific_univariate
     new_nicas: 1
     write_nicas_local: 1
     mask_check: 1
     resol: 6.0
-    network: 0 # Not re-implemented yet
```
Comment (guillaumevernieres) [inline test/testinput/dirac_soca_mask.yml:27]: @benjaminmenetrier , does that mean that this option is gone for ever?

Comment (benjaminmenetrier) [reply test/testinput/dirac_soca_mask.yml:27]: Probably, until I find some time to re-implement it.

**Comment by benjaminmenetrier** [comment]
This PR can be merged now.

**Review by shlyaeva — APPROVED** [review]
Changes in code and yaml look good, changes in references look reasonable to me (with the caveat that I don't know much about what's expected :))

**Comment by shlyaeva** [comment]
@kbhargava, @guillaumevernieres note: soca develop would be broken until this PR is merged.

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Comment by kbhargava** [comment]
> @kbhargava, @guillaumevernieres note: soca develop would be broken until this PR is merged.

Thanks @shlyaeva . I am reviewing it right now and will merge this once done.

**Review by kbhargava — APPROVED** [review]
Looks fine. Thanks @benjaminmenetrier and @shlyaeva

**Pull request merged**